ext/session/session.c

```diff
@@ -1704,6 +1704,14 @@ static PHP_FUNCTION(session_set_cookie_params)
 		zend_string *key;
 		zval *value;
 
+		if (path) {
+			path = NULL;
+			domain = NULL;
+			secure_null = 1;
+			httponly_null = 1;
+			php_error_docref(NULL, E_WARNING, "Arguments passed after the options array are ignored");
```

Comment:
I don't think they should be ignored. This should be a return false condition at least.